tfs/errors.py

```diff
@@ -5,6 +5,102 @@
 Errors that can be raised during the handling of **TFS** files.
 """
 
+from pathlib import Path
+
+# ----- Main Exception to Inherit From ----- #
```

Comment:
**FEATURE**

All of these exceptions correspond to scenarios where _we were already raising an error_. The errors are just now more specific, and have tailored error messages.